README.md

```diff
@@ -15,11 +15,37 @@ LPWC also supports irregular time intervals between time points collected in bio
 
 Installation
 ------------
-The first major release of LPWC will be added to CRAN.
-Until then, the easiest way to install LPWC is as follows:
+Prior to analyzing your data, the R package needs to be installed.
+
+The easiest way to install LPWC is through R:
```

Comment:
Should we say `through CRAN` instead of `through R`?

Reply:
Let's do CRAN